src/qml/qgismobileapp.qml

```diff
@@ -16,7 +16,6 @@
  ***************************************************************************/
 
 import QtQuick 2.11
-import QtQuick.Controls 1.4 as Controls
```

Comment:
Cool!!!!

Reply:
That was the issue. The `menuItem` from 1.4 did not work. 
The `Button.qml` used the Controls 1.4 as well. Possibly we could replace it there as well. But for now it does not lead to problems.